templates/api/knowledge-panels/environment/ecoscore/ecoscore_extended.tt.json

```diff
@@ -0,0 +1,69 @@
+{
+    "level" :"info",
+    "topics": [
+        "environment"
+    ],
+    "title_element": {
+        "title": "Eco-Score extended",
+        "subtitle": "[% lang('categories_s') FILTER ucfirst %][% sep %]: [% panel.agribalyse_category_name.dquote %]",
+        "type": "grade",
+        "grade": "[% panel.agribalyse_grade %]",
+    },
+    "elements": [
+        {
+            "element_type": "text",
+            "text_element": {
+                "text_type": "summary",
+                "html": `eco-score extended panel
+
+                ecoscore_extended_data: {
+                    impact: {
+                    average_total_used_mass: 100.084281307652,
+                    calculation_time: 56.4590351581573,
+
+                    testing access:
+
+                    calculation_time: [% product.ecoscore_extended_data.impact.calculation_time %]
+
+                    ecoscore_extended_data_more_precise_than_agribalyse: [% panel.ecoscore_extended_data_more_precise_than_agribalyse %]
+
+                    `
+            }
+        },
+        {
+            "element_type": "table",
+            "table_element": {
+                "id": "ecoscore_lca_impacts_by_stages_table",
+                "table_type": "percents",
+                "title": "[% lang('ecoscore_impact_detail_by_stages') %]",
+                "columns": [
+                    {
+                        "text": "Agribalyse",
+                        "type": "text",
+                    },
+                    {
+                        "text": "Estimator",
+                        "type": "text",
+                    }
+                ],
+                "rows": [
+                    [% FOREACH ingredients IN product.ecoscore_extended_data.impact.max_confidence_recipe %]
+                    {
+                        "id": "[% step %]",
```

Comment:
where does step come from ?

Reply:
In my test, this is alway empty

Reply:
this can be ignored for now, it's just a copy paste that will be removed